### ISSUE TITLE
Lean Overlay is not being removed.

### ISSUE THREAD

**Issue by jhernandezb** [react-materialize/react-materialize#141]
Basically the overlay (black screen) is removed when you click outside the modal or you press the `ESC` key , but if you are working for example with react-router this is not the case, since the componentWillUnmount only removes the modal and not the overlay

Few options
In [Overlay.js](https://github.com/react-materialize/react-materialize/blob/master/src/Overlay.js#L5)

```
$('#' + this.overlayID).closeModal();

// or  find the id and remove the node manually.
$('#' + this.overlayID).data('overlay-id');
```

Any other ideas? I can submit a PR.

**Comment by isaiah** [comment]
@jhernandezme PR is always welcome :)

**Comment by alextrastero** [comment]
Solved in `react-materialize@2.0.0`
